l10nzh.properties

```diff
@@ -2,15 +2,23 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=10.4
-publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5,10.6,10.7
+publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5,10.6,10.7,24.12
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+24.12.description=Support SonarQube 24.12
+24.12.sqs=[10.8,LATEST]
+24.12.sqcb=[24.12,LATEST]
+24.12.sqVersions=[24.12,LATEST]
+24.12.date=2024-12-06
+24.12.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-24.12
+24.12.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-24.12/sonar-l10n-zh-plugin-24.12.jar
+
 10.7.description=Support SonarQube 10.7
-10.7.sqs=[10.8,LATEST]
-10.7.sqcb=[24.12,LATEST]
-10.7.sqVersions=[10.7,10.7]
+10.7.sqs=[10.7,10.7.*]
+10.7.sqcb=[10.7,10.7.*]
+10.7.sqVersions=[10.7,10.7.*]
```

Comment:
There's no need for a range here, especially since 10.8 has been released, so we know there won't be a 10.7.1. This will work syntactically, but you could make this just `10.7` (with no brackets) if you wanted.